ipa-core/src/helpers/transport/in_memory/transport.rs

```diff
@@ -268,7 +274,7 @@ impl Debug for InMemoryStream {
 }
 
 pub struct Setup<I> {
-    identity: I,
+    pub identity: I,
```

Comment:
```suggestion
    identity: I,
```
Seems like `pub` is not needed here?